test/Rings-conformance-tests.jl

```diff
@@ -21,6 +21,113 @@ function equality(a::T, b::T) where T <: AbstractAlgebra.NCRingElement
    end
 end
 
+function test_mutating_op_like_zero(f::Function, f!::Function, a)
+  A = deepcopy(a)
+  a = f!(a)
+  @test equality(a, f(A))
+  a = deepcopy(A)
+end
+
+function test_mutating_op_like_neg(f::Function, f!::Function, a, b)
+  A = deepcopy(a)
+  B = deepcopy(b)
```

Comment:
How about switching these for symmetry with the later `deepcopy` calls?
```suggestion
function test_mutating_op_like_neg(f::Function, f!::Function, A, B)
  a = deepcopy(A)
  b = deepcopy(B)
```